enrollment-server-api-model/src/main/java/com/wultra/app/enrollmentserver/api/model/onboarding/request/DocumentStatusRequest.java

```diff
@@ -15,7 +15,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.app.enrollmentserver.api.model.request;
+package com.wultra.app.enrollmentserver.api.model.onboarding.request;
```

Comment:
What about to introduce a new module `enrollment-server-onboarding-api-model`?

Reply:
I was about to do it and then I thought: "I am doing the minimal fix, they will have to fix this themselves"

I admit: Big motivation for leaving it up to you is that we also have "domain model" package... which probably also should be considered.

Reply:
Opps, _api-model_ depends on `domain-model`, another reason to split.